### PR TITLE
Imgaug

### PR DESCRIPTION
**Pull request by edmondac**
For INC1052044

`imgaug-0.4.0-foss-2019b-Python-3.7.4.eb`
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-power9
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('Shapely', '1.7.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/Shapely/'],
+        'checksums': ['e21a9fe1a416463ff11ae037766fe410526c95700b9e545372475d2361cc951e'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:29]:
```suggestion
```
We have https://bear-apps.bham.ac.uk/applications/Shapely/1.7.0-foss-2019b-Python-3.7.4/

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow', '6.2.1'),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('scikit-image', '0.16.2', versionsuffix),
+    ('OpenCV', '4.2.0', versionsuffix),
+    ('GEOS', '3.8.0', versionsuffix),
```
Comment (branfosj) [inline easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:19]:
```suggestion
    ('GEOS', '3.8.0', versionsuffix),
    ('Shapely', '1.7.0', versionsuffix),
```

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('Shapely', '1.7.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/Shapely/'],
+        'checksums': ['e21a9fe1a416463ff11ae037766fe410526c95700b9e545372475d2361cc951e'],
+    }),
+    ('imageio', '2.9.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:30]: Is https://bear-apps.bham.ac.uk/applications/imageio/2.6.1-foss-2019b-Python-3.7.4/ new enough instead here?

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb (hunk 2 above)
Comment (branfosj) [inline easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:19]: Is GEOS really needed? It looks an odd requirement.

Comment (edmondac) [reply easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:19]: It was in the upstream previous version... but I see what you mean...

Comment (edmondac) [reply easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb]: I've taken it away

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb (hunk 3 above)
Comment (edmondac) [reply easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:26]: Let's try it :-)

**Pull request merged**